packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -28,7 +28,7 @@ describe('App: Settings', () => {
   })
 
   describe('Cloud Settings', () => {
-    it('shows the projectId section when there is a projectId', () => {
+    it('shows the projectId section when there is a projectId and shows override from CLI', () => {
```

Comment:
Any reason we didn't just add a new test vs change the existing one?

Reply:
I was hesitant to add a new test because they're pretty slow at startup, and this was half covered by the existing test.